js/acquia_lift.debugger.js

```diff
@@ -0,0 +1,1885 @@
+/**
+ * Prevents parent element from scrolling when hovering over debugger elements.
+ */
+
+function preventDefault(e) {
+    // converts event for all browsers
+    e = e || window.event; 
+    if (e.preventDefault) e.preventDefault(); 
+    e.returnValue = false;
+}
+var scrollElement = function(e,elementID){
+    var scroll;
+    if ( e.wheelDelta ) { // IE and Opera and Chrome
+        scroll = e.wheelDelta; //IE
+        if(e.wheelDeltaY){
+            scroll = e.wheelDeltaY; //Opera and Chrome
+        }
+    }
+    else if (e.detail) { // Mozilla FireFox
+        scroll= -e.detail;
+    }
+    //sets scroll amount if element is a class
+    if(document.getElementsByClassName(elementID)[0]){ 
+        document.getElementsByClassName(elementID)[0].scrollTop -= scroll;
+    //sets scroll amount if element is ID
+    }else if (document.getElementById(elementID)){ 
+        document.getElementById(elementID).scrollTop -= scroll;
+    }
+    //prevents parent elements from scrolling.
+    preventDefault(e);
+    e.stopPropagation();
+}
+
+//checks if user is scrolling in the debugger tool
+document.getElementById('debugger').onmousewheel = function (e) {
+    //scroll only the debugger content if mouse is hovering in the debugger section
+    scrollElement(e,'debugger__content'); 
+    if(document.getElementById('debugger__autocomplete__dropdown')){ 
+        //scroll only the autocomplete dropdown if mouse is hover in that space.
+        document.getElementById('debugger__autocomplete__dropdown').onmousewheel = function(e){
+            scrollElement(e,'debugger__autocomplete__dropdown')
+        } 
+    }
+}
+
+var app = angular.module("debuggerModule", ['isteven-multi-select', 'autocomplete','angularResizable']);
```

Comment:
Let's have a comment delimiter here that makes it clear that this starts the angular code.